WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -176,6 +176,14 @@ import Reachability
         displayTitleViewOnly = true
     }
 
+    func configureForNoConnectionInPostList() {
```

Comment:
Having this kind of customization that's specific to one of the App's VCs in `NoResultsViewController.swift` isn't great, because it means we have some of the logic for the post list VC in one place, and part of in another place.

In my opinion this class should be mostly abstract and not have logic that's specific to any parent VC.

Considering there's already a `configure` method that allows us to configure all this, can we call that the post list customizations all live in `PostListViewController.swift`?

Reply:
Actually I just understood why you're not using the existing configure method.  I'll ping you over DM to suggest an alternative.